src/main/resources/db/changelog/changes/v5.1.1/update-marc-specification-bibliographic.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+  <changeSet id="MODQM-405@@update-marc-specification-bibliographic" author="phabas">
+    <preConditions onFail="MARK_RAN">
+      <sqlCheck expectedResult="1">
+        SELECT COUNT(id)
+        FROM marc_specification
+        WHERE field_tag = '008'
+          and record_type = 'MARC_BIBLIOGRAPHIC';
+      </sqlCheck>
+    </preConditions>
+
```

Comment:
Please add a comment section

Reply:
I'm not sure what do you mean, I added a xml style comment with some summary.